### PR TITLE
Added my project fibonacci-series.yml

### PR DESCRIPTION
**Pull request by fharookshaik**
Hi there!

I hope that my project will fit for the Up for grabs.

Please give any kind of feedback that you feel to improve my project.

Thank You

### CODE REVIEW

File: _data/projects/fibonacci-series.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - learn
+  - fibonacci-sequence
+  - programming-languages
+  - code-development
+  - open-source
+  - community-driven
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/fharookshaik/fibonacci_series/labels/help%20wanted
```
Comment (ritwik12) [inline _data/projects/fibonacci-series.yml]: @fharookshaik As per the name in line above the link should be https://github.com/fharookshaik/fibonacci-series/labels/up%20for%20grabs

File: _data/projects/fibonacci-series.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: fibonacci-series
+desc: This repository is to write program the famous Fibonacci series in as many different programming languages as we can.
+site: https://github.com/fharookshaik/fibonacci_series
```
Comment (ritwik12) [inline _data/projects/fibonacci-series.yml]: @fharookshaik I believe you changed the Repo name to https://github.com/fharookshaik/fibonacci-series. please correct.

```suggestion
site: https://github.com/fharookshaik/fibonacci-series
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by fharookshaik** [comment]
Required changes have been made.

**Comment by ritwik12** [comment]
@fharookshaik Thanks!!

**Pull request merged**